src/main/resources/CodyBundle.properties

```diff
@@ -165,6 +165,7 @@ action.sourcegraph.openFindPopup.text=Find with Sourcegraph...
 action.sourcegraph.openFindPopup.description=Search all your repos on Sourcegraph
 action.sourcegraph.login.text=Log in to Sourcegraph
 action.sourcegraph.login.description=Log in to Sourcegraph
+action.sourcegraph.disabled.description=Log In to Sourcegraph to enable Cody features
```

Comment:
Use consistent capitalisation. It should be either sentence case or [title case](https://en.wikipedia.org/wiki/Capitalization#Titles).